sdp.js

```diff
@@ -551,6 +551,41 @@ SDPUtils.parseMsid = function(mediaSection) {
   }
 };
 
+// SCTP Datachannels
+// parses either the older sctpmap format, or the newer sctp-port format
+SDPUtils.parseSctpMap = function(mediaSection) {
+  var sctpPort = SDPUtils.matchPrefix(mediaSection, 'a=sctp-port:');
+  if (sctpPort.length > 0) {
+    return {
+      number: sctpPort[0].substr(12),
+      protocol: 'webrtc-datachannel',
+      streams: '1024'
```

Comment:
We should not hand out this information at all. SCTP will negotiate and supersede this value in its handshake which is probably why it has been removed.